.changeset/config.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [
+    [
```

Comment:
Maybe a better workaround for this would be to propose an addition to Changesets that makes lockstep versioning easier. We could add `lockstepped` flag to config which would automatically use monorepo packages as "linked" packages. 

This way we don't need to do anything funky. What do you think?

Reply:
Yeah that'd definitely be a nicer solution!

Reply:
I'm not the only person who had this idea https://github.com/atlassian/changesets/issues/364